src/projectile.cpp

```diff
@@ -108,22 +108,22 @@ void apply_ammo_effects( const tripoint &p, const std::set<std::string> &effects
     if( effects.count( "NAPALM" ) > 0 ) {
         g->explosion( p, 4, 0.7, true );
         // More intense fire near the center
-        for( auto && pt : g->m.points_in_radius( p, 1, 0 ) ) {
+        for( auto &&pt : g->m.points_in_radius( p, 1, 0 ) ) {
```

Comment:
It's not an issue of astyling or something, but why use rvalue references here? Can't they just be replaced with `const auto &pt`?